examples/PointSimulatorLive.ts

```diff
@@ -0,0 +1,172 @@
+import {
```

Comment:
this whole file is basically a copy of PointSimulator but with the update state function implemented.